java/client/src/org/openqa/selenium/devtools/network/model/ResourceTiming.java

```diff
@@ -54,6 +58,32 @@
 
   private Double receiveHeadersEnd;
 
+  public ResourceTiming(Double requestTime, Double proxyStart, Double proxyEnd,
```

Comment:
For fields that are not optional, prefer `double` to `Double`. That means it's impossible to create an instance in an incorrect state, and it saves you needing to do null checks.

Reply:
done